modules/videoio/misc/plugin_ffmpeg/CMakeLists.txt

```diff
@@ -5,7 +5,10 @@ include("${OpenCV_SOURCE_DIR}/cmake/OpenCVPluginStandalone.cmake")
 
 # scan dependencies
 set(WITH_FFMPEG ON)
-set(OPENCV_FFMPEG_SKIP_BUILD_CHECK ON)
+
+if(EMSCRIPTEN AND NOT DEFINED OPENCV_FFMPEG_SKIP_BUILD_CHECK)
```

Comment:
You still trying to modify scripts for **standalone plugin**.

OpenCV.JS doesn't use them at all.
Such plugins are build separately without OpenCV.

See here:

https://github.com/opencv/opencv/blob/4b6047e746f06ce3e595c886cf6c0266498c6a67/modules/videoio/CMakeLists.txt#L2-L4

---

You can remove/rename this file and ensure that it is not used - build OpenCV locally without any changes.

Please verify patch locally before submitting them.